.gitattributes

```diff
@@ -0,0 +1,23 @@
+# Basic .gitattributes for a python repo.
```

Comment:
I'd also recommend setting a default for text file EOLs to LF

Reply:
smth like `* text=auto eol=lf` (need to check the syntax tho)

Reply:
@RootenberG you can see examples [here](https://github.com/alexkaratarakis/gitattributes/search?q=eol&unscoped_q=eol)